views/apartments/apt.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
consider renaming this file to `show.erb` - this is a convention we'll see this week and next in rails